buildSrc/src/main/groovy/io.micronaut.build.internal.liquibase.test-suite-module.gradle

```diff
@@ -0,0 +1,32 @@
+plugins {
+    id("groovy")
+    id("io.micronaut.library")
```

Comment:
Please, remove dependencies to our [external Gradle plugins](https://github.com/micronaut-projects/micronaut-serialization/pull/408#issuecomment-1477383153).

Reply:
Like this? https://github.com/micronaut-projects/micronaut-liquibase/pull/343/commits/d042791b7d8dc18ef0d8408a9b300b423661e41e

Reply:
Hmmm...  Not sure the jackson tests are running 🤔